Homework-june-13/Daniel_Mourad/create-ec2.sh

```diff
@@ -0,0 +1,222 @@
+#!/bin/bash
+
+set -e
+
+ResourceIds="available_resource_ids.txt"
+vpcId=
+subnetId=
+internetGatewayId=
+routeTableId=
+securityGroupId=
+sshKeyName="ec2-key"
+sshKey="ec2-key.pem"
+instanceId=
+instancePublicIp=
+
+# creates vpc with 172.22.0.0/16 cidr block and sends vpc id to ResourceIds file
+function createVPC () {
+	echo "Creating VPC with 172.22.0.0/16 CIDR Block..."
+	aws ec2 create-vpc \
+		--cidr-block 172.22.0.0/16 \
+		--query Vpc.VpcId \
+		--output text > $ResourceIds && \
+	echo "VPC is created"
+	vpcId=$(cat $ResourceIds | head -1)
+}
+
+# creates subnet with 172.22.22.0/24 cidr block and sends subnet id to ResourceIds file
+function createSubnet () {
```

Comment:
write a better query to get exact Subnet ID
Dont create the additional file, use variable directly